### ISSUE TITLE
Provide Api blueprint parser api as alternative to use drafter locally.

### ISSUE THREAD

**Issue by fesor** [pixelfusion/blueman#28]
apiblueprint.org has an API, so maybe this library should provide optional way to use it instead of requiring drafter?

Here is an example of client:

https://github.com/fesor/json-schema-loader/blob/master/src/Apiary/Parser/RemoteApiBlueprintParser.php

**Comment by kielabokkie** [comment]
@fesor Yep, that would be a nice enhancement. Happy to accept a PR for it if someone can implement it before I get to it.

**Comment by hendrikmaus** [comment]
I think the actual parsing shouldn't be integrated with blueman. It is another build step in a pipeline before running blueman.
